MachineLearningPrototype/data/JenkinsData/(sample)openjdk-tests-1820.txt

```diff
@@ -13,17 +13,17 @@ Running on test-packet-ubuntu1604-x64-2 in /home/jenkins/workspace/Test_openjdk1
 [Pipeline] checkout
 No credentials specified
 Cloning the remote Git repository
-Cloning repository https://github.com/AdoptOpenJDK/openjdk-tests.git
+Cloning repository https://github.com/adoptium/aqa-tests.git
```

Comment:
For this file, you can change the file name to "(sample)aqa-tests-1820.txt", but leave the content as is, since this is the origin content from git issues. Same for the file under GitHubData folder too. Thanks.

Reply:
Changed the file names and restored the original contents.